src/main/resources/static/shiba.js

```diff
@@ -78,4 +78,8 @@ var noneOfTheAbove = (function() {
 $(document).ready(function () {
     followUpQuestion.init();
     noneOfTheAbove.init();
+    $("#page-form").submit(function () {
+        $("#form-submit-button").addClass("button--disabled");
+        return true;
+    });
```

Comment:
Originally I think the idea was to block just that final submit but I think it makes sense to target everything. In that case it'd be nice to figure out how to do this on all our non-standard pages too. I noticed it didn't work for `pages/whereToSendMail` for example.

Reply:
Yeah that's a good call out! The only challenge with that is that we _don't_ want to disable submit button for uploading documents on the upload documents page, but probably not too hard to work around that.